app.py

```diff
@@ -41,7 +41,8 @@
 
 _LOGGER = logging.getLogger('thoth.package_releases')
 PYPI_RSS_UPDATES = 'https://pypi.org/rss/updates.xml'
-PROMETHEUS_PUSH_GATEWAY = os.getenv('PROMETHEUS_PUSH_GATEWAY')
+push_gateway_host = os.getenv('PROMETHEUS_PUSHGATEWAY_HOST')
+push_gateway_port = os.getenv('PROMETHEUS_PUSHGATEWAY_PORT')
```

Comment:
These are global variables not exposed out of module - so they should be named uppercase, starting with underscore.